### PR TITLE
remove or revise logical defs, addresses #4915

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
- remove logical def for HP_0001166 Arachnodactyly 
- revise logical def for HP_0001626 Abnormality of the cardiovascular system
- remove logical def for HP_0001842 Foot acroosteolysis
- remove logical def from HP_0007519 'Lack of subcutaneous fatty tissue'
- remove logical def from HP_0004387 Enterocolitis
- remove logical def from HP_0045060	Aplasia/hypoplasia involving bones of the extremities
- reclassified 'Dilatation of the renal pelvis' as a child of HP_0010944 'Abnormal renal pelvis morphology'
- removed logical def from HP_0040065 Abnormal morphology of bones of the upper limbs
- removed logical def from HP_0040066 Abnormal morphology of bones of the lower limbs

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
I made some additional comments on the ticket #4915

File: src/ontology/hp-edit.owl [hunk 1]
```diff
@@ -33750,7 +33749,7 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
...
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0001626> "The cardiovascular system consists of the heart, vasculature, and the lymphatic system."^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0001626> "Abnormality of the cardiovascular system"^^xsd:string)
-EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0001626> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/UBERON_0004535>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
+EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0001626> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/BFO_0000019> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/UBERON_0004535>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
```
Comment (matentzn) [inline src/ontology/hp-edit.owl]: Protege bug, careful here! (BFO quality instead of PATO!)

Comment (nicolevasilevsky) [reply src/ontology/hp-edit.owl]: ok- I think I fixed that, thanks @matentzn

**Comment by drseb** [comment]
@nicolevasilevsky @matentzn I think we now have conflicts, because I obsoleted classes (merging). What do you suggest is the best way forward now?

**Comment by drseb** [comment]
Sorry about this. I didn't think about this before committing.

**Comment by matentzn** [comment]
That was easy! When I saw the message yesterday I thought you broke master :D Since it was all on a branch, super unproblematic! Feel free to merge now.

**Comment by nicolevasilevsky** [comment]
thanks @drseb and @matentzn!

**Pull request merged**